packages/tests/generated-packages/src/lib/generated-package-json.spec.ts

```diff
@@ -37,11 +37,15 @@ describe('testsGeneratedPackageJson', () => {
   const packageJsonPaths = findPackageJsonPaths('dist/packages');
 
   it.each(packageJsonPaths)(
-    '%s should have a type field',
+    '%s should have no type field if it has a module and a main field',
```

Comment:
Shouldn't description state that non of all three should be true?

Reply:
Or the other way round. Non of the should be set

Reply:
We are checking that if the file has both a module and a main field, it can't have also a type field.